modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp

```diff
@@ -288,6 +292,13 @@ ParallelCoordinates::ParallelCoordinates()
     });
     selectedColorAxis_.onChange([&]() { recreateLines_ = true; });
 
+    resetHandlePositions_.onChange([&]() {
+        for (auto axis : axisVector_) {
+            axis->moveHandle(true, std::numeric_limits<double>::max());
+            axis->moveHandle(false, std::numeric_limits<double>::min());
```

Comment:
Probably wany lowest not min...

Reply:
Does it make any difference? As far as I understand I send the mouse Y position as arguments, which should never be negative anyways, right?

Reply:
But since the handles can be more or less outside of the window perhaps you're right and it would be better to use lowest...

I'll change it to lowest!